Neos.Neos/Classes/Fusion/Helper/CachingHelper.php

```diff
@@ -69,6 +69,25 @@ public function nodeTag($nodes)
         return $this->convertArrayOfNodesToArrayOfNodeIdentifiersWithPrefix($nodes, 'Node');
     }
 
+    /**
+     * Generate a `@cache` entry tag for a single node identifier. If a $contextNode is given the
+     * entry tag will respect the workspace hash.
+     *
+     * @param string $identifier
+     * @param null $contextNode
+     * @return string
+     *
+     */
+    public function nodeTagForIdentifier(string $identifier, $contextNode = null): string
```

Comment:
$contextNode should be annotated as NodeInterface. Comment should be adjusted also.

Reply:
Jip, totaly right! Adjusted it @daniellienert !